lms/djangoapps/learner_dashboard/programs.py

```diff
@@ -71,13 +75,90 @@ class ProgramDetailsFragmentView(EdxFragmentView):
     Render the program details fragment.
     """
 
+    @staticmethod
+    def get_program_discussion_configuration(program_uuid):
+        return ProgramDiscussionsConfiguration.objects.filter(
+            program_uuid=program_uuid
```

Comment:
would this fail if the config doesn't exist for a program?

Reply:
No. This will return None which is being handled in the method that uses this